plugins/inputs/webhooks/webhooks.go

```diff
@@ -0,0 +1,81 @@
+package webhooks
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+	_ "github.com/influxdata/telegraf/plugins/inputs/webhooks/webhooks_all"
+	"github.com/influxdata/telegraf/plugins/inputs/webhooks/webhooks_models"
+)
+
+func init() {
+	inputs.Add("webhooks", func() telegraf.Input { return NewWebhooks() })
+}
+
+type Webhooks struct {
+	ServiceAddress string
+
+	Webhook []WebhookConfig
+}
+
+type WebhookConfig struct {
```

Comment:
For the Number 1, we observed that all fields of the plugin `struct` are initialized.

This means that even if the rollbar configuration is missing, like the configuration above, we still get an initialized struct `RollbarConfig` (in `WebhookConfig`) with an empty field `path`. So the nil check doesn't work to know if the plugin is configured or not.